akhampariya/tagg
# Merge release to dev (#72)

* Merge Sprint 1 to Production (#69)

* update dependencies

* update readme

* update instructions

* fix .env file

* update git instructions

* update .env and fix migration issue

* update readme

* fix .env for mailtrap instructions

* update readme

* Sprint1 (#68)

* move manual donation request to drop-down

* separate url generator/business profile pages and update nav bar to access url page

* update readme for sprint 1 instructions

* put debugger info

* Update initial DB Seeder

* fix offset issue

* request pending error fix on business user dashboard

* added 'set my donation preferences' page

* test

* fix state

* fix state migration and seed

* Bug Fix: Create Password page for the new user

Changed the verbiage from 'Reset Password' to 'Create/Reset Password' and the button from 'Reset Password' to 'submit'

* Change email template to remove last name

* fix buttons and navigation

* fix string to text in migration

* Bug Fix: Added number formatting

Added number formatting for Requested amount being reflected on the Dashboard.

* fix email issue

* fixed create org routes

* Bug Fix: Close button

Removed the button for 'Close'

* single branch cloning instructions

* bug fix: Added number format

* Bug Fix: Added number formatting

* Bug Fix: Number Formatting

* Bug Fix: Number formatting

* More bug fixes

* More changes to number formatting

* gitignore

* create id

* b user

* fix email issue

* encrypt url

* NumberFormat added

* Fix - Donation request issue

* Update package dependency

* Add log viewer

* Add Log viewer

* Fix Auto focus

* 2/14

* Test seeds

* Test seeds

* Test seeds

* Test seeds

* Test seeds

* sql query

* Pricing template

* Change header text

* update pricing

* update pricing

* merge

* fix donationurl

* pending request count fix

* pkg update

* conflict fix

* remove lock file

* update pkg

* change git instructions

* add build check

* update pkg dependencies

* compile css
